docs/install.rst

```diff
@@ -165,10 +123,36 @@ Note the following:
 * Starting up the EC2 instance will take several minutes.
 * This has been tested with *Ubuntu Server 14.04 LTS*, but may work with other AMI types.
 * We don't have recommendations for sizing, but providing multiple processors even for
-  testing/experimentation.
+  testing/experimentation is suggested.
 * If you need to make additional changes to your ``docker-compose.yml``, you can ssh into the EC2 instance
-  and make changes.  ``docker-compose.yml`` and ``secrets.env`` will be in the default user's
+  and make changes.  ``docker-compose.yml`` and ``.env`` will be in the default user's
   home directory.
 * Make sure to configure a security group that exposes the proper ports. To see which
-  ports are used by which services, see `master.docker-compose.yml <https://github.com/gwu-libraries/sfm-docker/blob/master/master.docker-compose.yml>`_.
+  ports are used by which services, see `example.docker-compose.yml <https://github.com/gwu-libraries/sfm-docker/blob/master/example.docker-compose.yml>`_.
```

Comment:
Not sure if it makes a difference, but is there a reason it points to example.docker-compose.yml instead of example.prod.docker-compose.yml? Or if they're different, both, like you did in line 76.